### ISSUE TITLE
Search function non pure?

### ISSUE THREAD

**Issue by m4v15** [facn2/ApolloXII#46]
Really nice modularisation of all your code, but you're search function relies on the ```food``` variable to work, so testing it you would also have to require in food - maybe pass it as a second variable to the function (```JSONToSearchThrough```?) and then you've got a nice neat, easily testable function.

**Comment by m4v15** [comment]
relates #32

**Comment by yvonne-liu** [comment]
Yes! Thanks @m4v15 :sparkles: for clarifying on the pure function part too 👍
